roles/gluster_hci/tasks/hci_volumes.yml

```diff
@@ -1,5 +1,21 @@
 ---
 # Create volumes for gluster_hci
+
+- name: check if gluster_features_hci_cluster is set
+  fail:
+    msg: "gluster_features_hci_cluster is undefined"
```

Comment:
@Sheersha-jain can you change the message to:

Variable gluster_features_hci_cluster has to be defined.